lib/getAccessorsForSemantic.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = getAccessorsForSemantic;
+
+/**
+ * Iterates over the provided glTF meshes and calls the callback when a primitive is
+ * found that uses a particular semantic.
+ *
+ * @param {Object} gltf The glTF hierarchy
+ * @param {String} semantic Matches against the beginning of the semantic string for the primitives' attributes
+ * @param {Function} callback This gets called when a match is found with the arguments (gltf, primitive, matchedSemantic, accessorId)
+ * @param {Function} callback This gets called when the iteration completes
+ */
+function getAccessorsForSemantic(gltf, semantic, callback, done) {
+    var meshes = gltf.meshes;
+    for (var meshId in meshes) {
```

Comment:
As discussed offline, this didn't really make sense to do in practice, but I am going to change it so that is uses promises instead of a done callback.